packages/react-native-dom/ReactDom/views/Text/RCTShadowText.js

```diff
@@ -78,13 +119,7 @@ module.exports = (async () => {
           get: () => this.props[shadowPropName],
           set: (value) => {
             if (value != null) {
-              this.props[shadowPropName] = TEXT_PX_PROPS.includes(
-                shadowPropName
-              )
-                ? `${value}px`
-                : value;
-            } else {
-              this.props[shadowPropName] = "inherit";
+              this.props[shadowPropName] = value;
```

Comment:
The crazy amount of whitespace in the lineHeight text example is from this change